### DISCUSSION TITLE
Using email alias from email provider together with SimpleLogin?

### DISCUSSION THREAD

**Discussion by sefsturlo** [simple-login/app#1729] [Q&A]
1. Is using an email alias from your email provider different to using an email alias from another email alias service (ie any SimpleLogin competitors service)?

2. Can you clarify if using the _email alias feature from an email provider_ still creates problems when using it together with SimpleLogin?

Eg
Using email alias from SL competitor: 
Alias@SLAlias.com > Alias@AnotherServiceAlias.com > Real@EmailProvider.com
(using addition alias service in the example above is not recommended) 

Using email alias from email providers:
Alias1@SLAlias.com > Alias1@AnyEmailProvider.com > Real1@AnyEmailProvider.com
Alias2@SLAlias.com > Alias2@Proton.me > Real2@ProtonMail.com
(as shown above, is using email alias from your email provider possible?)

> Email aliases (or email forwarding) can be provided by multiple services, which are often incompatible. Some forwarding/alias email services also don't preserve email authentication such as SPF, DKIM, or DMARC, and their emails can be put into quarantine/spam or simply rejected.
> 
> Therefore, we recommend not using another email alias/forwarding service together with SimpleLogin. A SimpleLogin mailbox needs to be a final email address,
> 
> As some mailbox services rewrite the email headers when forwarding the email, which interferes with the SimpleLogin system, you won't be able to reply to a forwarded email or send emails from an alias.
> 
> Info from https://simplelogin.io/docs/getting-started/compatibility-email-alias/
> 

4. So using email alias/forwarding service together with SimpleLogin is not recommended but what about using the email alias from your email provider? Will this work without any issues? 

5. How can we check if email authentication such as SPF, DKIM, or DMARC is preserved?

6. How can we check our email provider doesn't rewrite the email headers when forwarding the email?

**Comment by fm** [comment]
1- In a nutshell, it's essentially the same - however, SL creates a brand new email when forwarding instead of preserving the original headers and just passing along the original message.

2- If the other alias provider and your email provider are the same _(eg: Fastmail's masked email service, or iCloud Hide my Email)_ generally there is no issue. However, if the other alias provider is not the same as your final email provider _(eg: SL -> AnonAddy -> iCloud)_, there will likely be issues. It is not recommended to chain alias providers and the net benefits are negligible. However your option two, using the Proton example, should not cause any issues as both the alias and the email box are on the same provider.

4- See 2) above

5- SimpleLogin creates an entirely new email when forwarding, the original authentication results will not matter to the final mailbox as the original sender does not appear anywhere in the 5321.From or 5322.From, so there should not be any issues with DMARC as long as you don't chain providers as mentioned in 2)

6- SimpleLogin creates an entirely new email when forwarding, so none of the original sender's headers appear. Currently there is no way to keep the original headers when using SL. However, if you're doing further forwarding after it lands at the destination, that will depend on your email provider and is outside the scope of SL or this forum.
